### DISCUSSION TITLE
Any plans for a new release?

### DISCUSSION THREAD

**Discussion by codygman** [nrdxp/nixdev#331] [General]
I would like to update to master of devos so I can use agenix but I cannot since overlay overriding is still broken.

I tried using devos in the develop branch with digga on the develop branch to get the fix for overlays:

https://github.com/divnix/digga/pull/55

I either did something wrong there, or that fix was not enough to fix overlays. My messages for two reasons:

- The devos team might not know that I or others are in this state
- To get any idea of when a new release may be cut so I can better verify if there is still an issue with overlays, especially the emacs overlay for me.

Thanks for all your volunteer work, and please don't take this as me trying to rush you. This is more so I can manage my own expectations and be sure the status is known.

**Comment by blaggacao** [comment]
There is currently some testing improvements done on the digga side of things. That might ultimately lead us to an always green develop and thereby mature us a lot towards stable.

We might even get rid of `develop` branches once they are always green.

I think if we push things a little we might be able to cut a new "release" to master within two weeks or so.

**Reply by benneti** [reply]
that sounds amazing, in particular with the new way to update pkgs
